.env.example

```
@@ -0,0 +1,15 @@
+# 애플리케이션 설정
+PROJECT_NAME=FastAPI Community
+VERSION=1.0.0
+API_V1_STR=/api/v1
+DEBUG=True
+
+
+# JWT 설정
+SECRET_KEY=your-super-secret-key-here
+ALGORITHM=HS256
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ALGORITHM key should go before the SECRET_KEY key

(UnorderedKey)